src/ott/geometry/costs.py

```diff
@@ -304,34 +304,46 @@ def pairwise(self, x: jnp.ndarray, y: jnp.ndarray) -> jnp.ndarray:
   def covariance_fixpoint_iter(
       self,
       covs: jnp.ndarray,
-      lambdas: jnp.ndarray,
-      rtol: float = 1e-2
+      weights: jnp.ndarray,
+      tolerance: float = 1e-4,
+      kwargs_sqrtm: Optional[Mapping[str, Any]] = None
```

Comment:
Why not use `**kwargs`?

Reply:
at that moment, this was used to avoid mixing `threshold` (which is used for `sqrtm` fixed point) and `tolerance` (for barycenter. Both can be defined though. However, we'll be running into issues if we want to open other parameters (such as `min_iterations`) since we have 2 imbricated fixed point loops (`sqrtm` and cov. barycenter).